core/Segment/SegmentExpression.php

```diff
@@ -301,11 +301,24 @@ protected function getSqlMatchFromDefinition($def, &$availableTables)
             }
         }
 
-        $this->checkFieldIsAvailable($field, $availableTables);
+        $columns = $this->parseColumnsFromSqlExpr($field);
+        foreach ($columns as $column) {
+            $this->checkFieldIsAvailable($column, $availableTables);
+        }
 
         return array($sqlExpression, $value);
     }
 
+    /**
+     * @param string $field
+     * @return string[]
+     */
+    private function parseColumnsFromSqlExpr($field)
+    {
+        preg_match_all('/\b([a-zA-Z0-9_`]+\.[a-zA-Z0-9_`]+)\b/', $field, $matches);
```

Comment:
@diosmosis not sure if on purpose... when field is eg "`log_action`.`type`" then

`preg_match_all('/\b([a-zA-Z0-9_`]+\.[a-zA-Z0-9_`]+)\b/', '`log_action`.`type`', $matches);`

returns this:
![image](https://user-images.githubusercontent.com/273120/49611886-e5af8300-fa07-11e8-895b-2eb71b4ac90e.png)
also see https://3v4l.org/0v32G 
not sure if the field can be like this.

Reply:
Updated to better handle backticks (it's not super intelligent, but should work given this is for plugin devs).